Runtime/JSMarshalAsAttribute.cs

```diff
@@ -0,0 +1,37 @@
+
+using System;
+
+namespace NodeApi;
+
+/// <summary>
+/// Specifies how .NET types are marshalled to and from JavaScript values.
+/// </summary>
+[AttributeUsage(
```

Comment:
>AttributeUsage [](http://example.com/codeflow?start=1&length=14)

Should we also add fields?

Reply:
Later, if and when we support marshalling fields.

Fields are not supported now, but there's no reason they can't be marshalled in almost the same way as properties, along with an optimization for `const` fields. I don't think it's a priority now though.